File: __Template/Py/FizzBuzz.py
def fizzBuzz(num):
    for item in range(num):
        if (item % 15 == 0):
            print("FizzBuzz")
        elif(item % 3 == 0):
            print("Fizz")
        elif(item % 5 == 0):
            print("Buzz")
        else:
            print(item)


fizzBuzz(20)

File: __Template/JS/fizzBuzz.js


function fizzBuzz(inum)
{
    for(var num=0;num<inum;num++)
    {
        if(num % 15 == 0)
            console.log("FizzBuzz");
        else if (num % 3 == 0)
            console.log("Fizz");
        else if (num % 5 == 0)
            console.log("Buzz");
        else
            console.log(num);
    }
}
fizzBuzz(20);

